### PR TITLE
Updated requirements.txt

### PR DESCRIPTION
**Pull request by diangeli**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -5,10 +5,22 @@
...
 pre-commit==2.21.*
 isort==5.11.*
 python-dotenv>=0.5.1
+
+# extra requirements
+omegaconf==2.3.0
+pytorch-lightning==1.8.6
+timm==0.6.12
+
+# pytorch requirements
```
Comment (GustavHansen99) [inline requirements.txt:20]: What does the "+cu117"? do?

**Review by GustavHansen99 — APPROVED** [review]
LGTM

File: requirements.txt (hunk 1 above)
Comment (diangeli) [reply requirements.txt:20]: It loads the pytorch version with cuda that we use to train and evaluate with gpu.

Comment (nikarav) [reply requirements.txt:20]: This version will most likely work with `google cloud`, but we can change it later if it doesn't

**Review by nikarav — APPROVED** [review]

**Pull request merged**